hack/tools/extractcheck/README.md

```diff
@@ -0,0 +1,5 @@
+This directory contains a linter for checking SSA Extract calls.
+
+Please refer to
+https://kubernetes.io/blog/2021/08/06/server-side-apply-ga/#using-server-side-apply-in-a-controller
+for SSA Extract model.
```

Comment:
That mentions `ExtractDeployment`, but not why that function should not be used. Some more information in this README is needed before it answers that.